install_scripts/java.sh

```diff
@@ -0,0 +1,5 @@
+###
+# Java
+###
+
+apt-get -y install openjdk-7-jre
```

Comment:
Any particular reason to use JDK 7 instead of JDK 8?  The Fedora base box has a [JDK 8 script](https://github.com/fcrepo4-exts/fcrepo4-vagrant-base-box/blob/master/install_scripts/java.sh) you can crib from.